Lib/email/headerregistry.py

```diff
@@ -245,13 +245,16 @@ def fold(self, *, policy):
         the header name and the ': ' separator.
 
         """
-        # At some point we need to only put fws here if it was in the source.
+        # At some point we need to put fws here iif it was in the source.
```

Comment:
iif stands for "if and only if" in this statement? [I've only ever seen iff used as an acronym for that.](https://en.wikipedia.org/wiki/If_and_only_if)

Reply:
Correct.